spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -233,49 +237,69 @@ Every Jakarta Data provider must support the following basic types within its pr
 | Custom enumerated types defined by user-written code.
 |===
 
+For example, the following entity class has three basic fields:
```

Comment:
The number might have just been a typo, or I'm missing something more subtle. All 5 fields of the entity have types that are listed under table of basic types.
```suggestion
For example, the following entity class has five basic fields:
```